app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -609,7 +670,8 @@ private void showErrorDialog(String errorMsg, String details) {
     public void playbackStateChanged(PlayerStatusEvent event) {
         boolean isPlayingPreview =
                 PlaybackPreferences.getCurrentlyPlayingMediaType() == RemoteMedia.PLAYABLE_TYPE_REMOTE_MEDIA;
-        viewBinding.stopPreviewButton.setVisibility(isPlayingPreview ? View.VISIBLE : View.GONE);
+        headerBinding.stopPreviewButton.setVisibility(isPlayingPreview ? View.VISIBLE : View.GONE);
+        headerBinding.playLayout.setVisibility(isPlayingPreview ? View.VISIBLE : View.GONE);
```

Comment:
This also changes the visibility of the "alternate urls" spinner, which is unrelated to playing